src/js/helpers.test.js

```diff
@@ -32,3 +29,85 @@ describe('FileSize Component', () => {
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe('getFormattedSize function', () => {
+  it('converts correctly', () => {
+    expect(getFormattedSize()).toEqual('0 Bytes');
+    expect(getFormattedSize(null)).toEqual('0 Bytes');
+    expect(getFormattedSize(0)).toEqual('0 Bytes');
```

Comment:
I guess only one scenario left: 
`expect(getFormattedSize(31)).toEqual('31 Bytes');`
just being mean. sorry.
feel free to disregard, since I am not sure getFormattedSize respects this.

Reply:
good point, except it will return `'31.00 Bytes'` - but I won't care for sub byte size 🤷‍♂ 😁